### PR TITLE
147 add labels to create project page and write them to system card on create project 

### LINKED ISSUE #147: Add labels to "Create Project page" and write them to system card on "create project"
In this task, we will add new fields to the create project page form. In this extended form, users can also select the labels that we also have in the decision tree. To get these labels, users can use the ai-decisiontree form.
See Clickup doc: https://app.clickup.com/9015737202/v/dc/8cp2evj-1815/8cp2evj-2015

Notes: 
- [x] Add a new field to the System card 
   - in which we store
      - the labels selected by the user (https://app.clickup.com/9015737202/v/dc/8cp2evj-515/8cp2evj-2275) 
      - we should reserve space for:
         - the path through the decision tree, this path contain the numbers but also *include the labels encountered along the path*.
         - the version of the decision tree
  - we should choose a name for the field, options are:
     - a risk_opensource_role_impact_profile
     - ai_act_profile
     - algorithmframework_profile
     - aisystem_profile field
  - we merge the publication category into the appropriate label
- don't forget to also adjust the system card schema
- [ ] #202 -> Make a task in the task-registry from the decision tree (but don't get the labels from there, get the labels from ai-act-decisiontree) RIGHT NOW WE WILL HARDCODE THIS 
- [ ]  Get requests from elsewhere are done in the following task #168
- [ ]  We translate labels in the following task #166
- [ ]  Embedding of the decision tree is done in the following task: #165

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# Description

Implements elements in the create project form to select labels from the AI Act Profile.

NOTE THE FOLLOWING:
1. The values of the labels are as for now all in Dutch. This is because the for now the values in the system card of these fields are all in Dutch. There is NO translation for these fields as for now. The category identifiers are translated.
2. The label fields are not required to be set by the user.
3. I could not make the dropdown select boxes smaller. This is implement for text fields, but apparently not for drop down selections (as far as I could tell).

Resolves #147

## Checklist

Please check all the boxes that apply to this pull request using "x":

-   [x] I have tested the changes locally and verified that they work as expected.
-   [x] I have followed the project's coding conventions and style guidelines.
-   [x] I have rebased my branch onto the latest commit of the main branch.
-   [x] I have squashed or reorganized my commits into logical units.
-   [x] I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

Screenshot 2024-09-24 at 15 58 09

### CODE REVIEW

File: tests/schema/test_schema_system_cards.py [hunk 1]
```diff
@@ -13,8 +13,9 @@ def setup() -> SystemCard:
 def test_get_system_card(setup: SystemCard) -> None:
     system_card = setup
     expected: dict[str, str | list[Any] | None] = {
+        "schema_version": "0.1a9",
```
Comment (laurensWe) [inline tests/schema/test_schema_system_cards.py:16]:
```suggestion
        "schema_version": "0.1a10",
```

File: tests/schema/test_schema_system_cards.py [hunk 2]
```diff
@@ -25,8 +26,8 @@ def test_get_system_card(setup: SystemCard) -> None:
 def test_system_card_update(setup: SystemCard) -> None:
     system_card = setup
     expected: dict[str, str | list[Any] | None] = {
+        "schema_version": "0.1a9",
```
Comment (laurensWe) [inline tests/schema/test_schema_system_cards.py:29]:
```suggestion
        "schema_version": "0.1a10",
```

**Comment by laurensWe** [comment]
@ChristopherSpelt rebase (if you were only a dependabot xd)

File: tests/services/test_projects_service.py [hunk 3]
```diff
@@ -49,7 +49,16 @@ def test_create_project():
...
-    project_new = ProjectNew(name=project_name, instruments=[])
+    project_new = ProjectNew(
+        name=project_name,
+        instruments=[],
+        type="project_type",
+        open_source="project_open_source",
+        publication_category="project_publication_category",
+        systemic_risk="project_systemic_risk",
+        transparency_obligations="project_transparency_obligations",
+        role="projet_role",
```
Comment (anneschuth) [inline tests/services/test_projects_service.py:60]:
```suggestion
        role="project_role",
```

File: tests/services/test_projects_service.py [hunk 4]
```diff
@@ -49,7 +49,16 @@ def test_create_project():
     projects_service.instrument_service.fetch_instruments.return_value = [default_instrument()]  # type: ignore
 
     # When
-    project_new = ProjectNew(name=project_name, instruments=[])
+    project_new = ProjectNew(
+        name=project_name,
+        instruments=[],
+        type="project_type",
+        open_source="project_open_source",
+        publication_category="project_publication_category",
```
Comment (anneschuth) [inline tests/services/test_projects_service.py:57]: Wouldn't this be more logical?

```suggestion
    project_new = ProjectNew(
        name=project_name,
        instruments=[],
        ai_act_profile={
            "type": "project_type",
            "open_source": "project_open_source",
            "publication_category": "project_publication_category",
            ...
```

Comment (ChristopherSpelt) [reply tests/services/test_projects_service.py:57]: This object is used to parse the data that is filled in the create project form. It matches on keys from the form. That's why it's implemented like this now. Changing it to your suggestion would be possible I think, but we need to change the way to form passes the data.

Comment (anneschuth) [reply tests/services/test_projects_service.py:57]: Ok, sounds this is closer to the form than I'd realized. Not neccessary for me.

**Review by laurensWe — APPROVED** [review]
LGTM :)

**Pull request merged**